app/src/main/java/com/wednesday/bankingdetails/data/FavouriteBankEntity.kt

```diff
@@ -0,0 +1,12 @@
+package com.wednesday.bankingdetails.data
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "favourite_banks")
+data class FavouriteBankEntity(
```

Comment:
why do we need a separate model for the bank and the favorite bank entity? Use the same class.